drivers/rptun/Kconfig

```diff
@@ -38,11 +45,4 @@ config RPTUN_PM
 		goto RAM-retention mode, can't access from another CPU.
 		So, we provide this method to resolve this.
 
-config RPTUN_PING
```

Comment:
why moved ? doesn't this feature depend on CONFIG_RPTUN=y?

Reply:
@raiden00pl we are restructure rptun to support non share memory transport(e.g. SPI/UART). The common part could be shared with them will be moved to driveers/rpmsg, ping will be part of the moving. After the reorganization, nuttx rptun/rpmsg driver framework will much like Linux kernel:
https://github.com/torvalds/linux/tree/master/drivers/rpmsg

Reply:
OK, that make sense